src/UI/Root/Root.test.tsx

```diff
@@ -63,15 +67,26 @@ function setup() {
   );
 
   const component = (
-    <MemoryRouter initialEntries={["/"]}>
-      <StoreProvider store={store}>
-        <DependencyProvider
-          dependencies={{ ...dependencies, queryResolver, environmentHandler }}
-        >
-          <Root />
-        </DependencyProvider>
-      </StoreProvider>
-    </MemoryRouter>
+    <QueryClientProvider client={queryClient}>
+      <MemoryRouter initialEntries={["/"]}>
+        <StoreProvider store={store}>
+          <DependencyProvider
+            dependencies={{
+              ...dependencies,
+              queryResolver,
+              environmentHandler,
+              authHelper: {
```

Comment:
Why wouldn't it be part of the mocked dependencies ? You'd avoid the need to repeat importing the defaultAuthContext all over the place.

Reply:
it all over the place as the commandResolvers require them, but I added it anyway, because I forgot it there